source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -1115,6 +1140,26 @@ clear the pool again. This situation is possible if the pool is cleared by the
 background thread after it encounters an error establishing a connection, but
 the ServerDescription for the endpoint was not updated accordingly yet.
 
+Why does the pool need to support interrupting in use connections as part of its clear logic?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
minor: the tilde and the previous line should have the same length?

Reply:
done